cms/djangoapps/contentstore/views/course.py

```diff
@@ -516,18 +519,18 @@ def format_library_for_view(library):
     return render_to_response('index.html', {
         'courses': courses,
         'in_process_course_actions': in_process_course_actions,
-        'libraries_enabled': LIBRARIES_ENABLED,
```

Comment:
Is there a reason you moved all these dictionary entries around? It makes it hard to tell what you changed.